### ISSUE TITLE
augmentConceptFiles - conceptAggregates issue

### ISSUE THREAD

**Issue by alondhe** [OHDSI/AresIndexer#40]
On this line: https://github.com/OHDSI/AresIndexer/blob/main/R/AugmentConceptFiles.R#L42

I'm seeing a dplyr warning thrown:

```
Warning message:
There was 1 warning in `filter()`.
ℹ In argument: `!is.na(CONCEPT_ID) && FAILED == 0`.
Caused by warning in `!is.na(CONCEPT_ID) && FAILED == 0`:
! 'length(x) = 3704 > 1' in coercion to 'logical(1)' 
```

I think a single ampersand should be used, per the dplyr documentation: "If multiple expressions are included, they are combined with the & operator."

Changing to a single ampersand allows this line to complete without error and produce the conceptAggregates data frame.

**Comment by alondhe** [comment]
One ampersand allows for a vector result, whereas two ampersands evaluates left to right, examining only the first element of each vector.

**Comment by fdefalco** [comment]
fixed in [7bcaa9a](https://github.com/OHDSI/AresIndexer/commit/7bcaa9a56af7a1b4f73e50ea8901dbd22830031b)
